src/domains/ioweb-common/03_storage.tf

```diff
@@ -60,4 +63,107 @@ resource "azurerm_storage_container" "spid_logs" {
   name                  = "spidlogs"
   storage_account_name  = module.spid_logs_storage.name
   container_access_type = "private"
+}
+
+
+######################
+# Immutable SPID LOGS Storage
+######################
+module "immutable_spid_logs_storage" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3//storage_account?ref=v7.32.1"
+
+  name                          = replace(format("%s-imm-spid-logs-st", local.project), "-", "")
```

Comment:
ditto

Reply:
Done in [fix based on comments](https://github.com/pagopa/io-infra/pull/778/commits/1e4268778c5a0b35cfe764917907553e84ffb6de)